### PR TITLE
[CELEBORN-1232] Add Menu to Celeborn Web

### PR DESCRIPTION
**Pull request by tiny-dust**
### What changes were proposed in this pull request?

image
image
image
image
image

### Why are the changes needed?

Configure the following menu and routing files for the Web UI:

- Overview
- Master
- Worker
- Application
- Tenant

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Local test.

### CODE REVIEW

File: web/src/layouts/layoutPage.vue [hunk 1]
```diff
@@ -17,6 +17,27 @@
...
+
+const menus = ref<MenuOption[]>([])
+
+const mapRouterToMenu = (routes:RouteRecordRaw[]) => {
+  if (routes) {
+    menus.value = routes.map(item => {
+      return {
+        label: item.meta?.title ?? '',
+        key: item.path,
+        path:item.path,
```
Comment (labbomb) [inline web/src/layouts/layoutPage.vue:33]: Missing space

File: web/src/views/master/index.vue [hunk 2]
```diff
@@ -17,10 +17,13 @@
 -->
 
 <script setup lang="ts">
+defineOptions({
+  name:'MasterView'
```
Comment (labbomb) [inline web/src/views/master/index.vue:21]: Missing space

**Review by labbomb — COMMENTED** [review]
@tiny-dust  thanks for your contribution, I left some comment,PTAL.

File: web/src/layouts/layoutPage.vue (hunk 1 above)
Comment (SteNicholas) [reply web/src/layouts/layoutPage.vue:33]: Could this format with `eslint --fix` via eslint-plugin-vue?

Comment (tiny-dust) [reply web/src/layouts/layoutPage.vue]: oh~ I'm sorry, I ran the `eslint` script. But I forgot to run the `format` script.

**Review by labbomb — APPROVED** [review]
LGTM. @tiny-dust thanks for your contribution.

**Review by SteNicholas — APPROVED** [review]

**Review by RexXiong — APPROVED** [review]
LGTM.

**Review by zwangsheng — APPROVED** [review]
LGTM, thanks

**Pull request closed without merging**